index.md

```diff
@@ -1,34 +1,25 @@
-Anbox Cloud makes it possible to run Android apps in the cloud and supports all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+Anbox Cloud enables running Android apps on any cloud platform at scale. It uses [LXD containers](https://linuxcontainers.org/) to run the nested Android containers and [Juju](https://charmhub.io/) for deployment in a cloud environment.
```

Comment:
Shouldn't this point to the Juju main website https://juju.is?
Also, I think we call them 'system containers', but we should check with @adglkh or @morphis
```suggestion
Anbox Cloud enables running Android apps on any cloud platform at scale. It uses [LXD system containers](https://linuxcontainers.org/) to run the nested Android containers and [Juju](https://juju.is) for deployment in a cloud environment.
```